packages/rOpenScPCA/tests/testthat/test-sweep-clusters.R

```diff
@@ -0,0 +1,123 @@
+suppressPackageStartupMessages(library(SingleCellExperiment))
+
+set.seed(2024)
+sce <- splatter::simpleSimulate(nGenes = 1000, verbose = FALSE) |>
+  scater::logNormCounts() |>
+  scater::runPCA(ncomponents = 10)
+
+
+test_that("sweep_clusters works as expected with defaults", {
+  sweep_list <- sweep_clusters(
+    sce,
+    nn = c(10, 15),
+    resolution = c(0.5, 1)
+  )
+
+  expect_equal(
+    length(sweep_list),
+    4
+  )
+
+  # check colnames one at a time
+  sweep_list |>
+    purrr::walk(
+      \(df) {
+        expect_setequal(
+          colnames(df),
+          c("cell_id", "cluster", "algorithm", "weighting", "nn", "resolution")
+        )
+
+        expect_equal(
+          df$cell_id,
+          colnames(sce)
+        )
```

Comment:
stylewise: put this and ones like it on one line?

```suggestion
        expect_equal(df$cell_id, colnames(sce))